prometheus-hanadb_exporter.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Wed Apr 29 15:00:34 UTC 2020 - Julien ADAMEK <julien.adamek@suse.com>
```

Comment:
I don't think we need to update the changelog for fixing a readme.md

Reply:
Yes, we need. The `README.md` file is part of the package and it's shipped, so we need to increase a new version and specify the changes to submit the new version

Reply:
I don't know but as the package is released in SLE, we need a bsc for pushing changes.
And if you have a bsc, he must appear in changelog from my "maintenance" process understanding.

Reply:
@arbulu89  ok . Good to know . I personally find it really un-usual but then so it is , resolving thx @arbulu89